### PR TITLE
bump chartjs to v2 and migrate current uses

### PR DESCRIPTION
**Pull request by taniki**

### CODE REVIEW

**Comment by taniki** [comment]
need some work around and tests. I have not yet find places where stacked bars are used.

**Comment by noirbizarre** [comment]
They weren't because Chart.js 1.X support was missing. Switching to Chart.js 2.x open up more charts possibilities. Next step will be to review charts usage in udata, propose some new ones, remove some useless ones, switch type on other...

**Comment by noirbizarre** [comment]
Can you add some screenshots ?

**Comment by taniki** [comment]
screenshot from 2017-12-19 16 45 14

@noirbizarre yep new charts are the next issue #1306  and consorts (:

File: js/components/dashboard/graphs.vue [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default {
             if (this.objectId) {
                 this.metrics.fetch({
                     id: this.objectId,
-                    start: moment().subtract(12, 'days').format('YYYY-MM-DD'),
+                    start: moment().subtract(30, 'days').format('YYYY-MM-DD'),
```
Comment (noirbizarre) [inline js/components/dashboard/graphs.vue]: Maybe define this as a constant for easier future reading/maintenance/changes

**Pull request merged**